index.html

```diff
@@ -0,0 +1,447 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="desktop-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
```

Comment:
- [ ] since the previous reviewer didn't mention this they are therefore optional.. the nav bar links are working perfectly but it has one little issue. let's make it look like the original Figma design. 
your design 👇🏾 ;
![a (2)](https://user-images.githubusercontent.com/65298048/169578047-47112493-d25b-4fe2-9e2a-a9529a1d6985.png)
--------
`Figma` design 👇🏾 
![b (2)](https://user-images.githubusercontent.com/65298048/169578202-3cd6ff3d-3e57-4064-9ba0-ccf32eb2bfb9.png)